kit/brain_board/wifi.md

```diff
@@ -3,63 +3,57 @@ layout: page
 title: WiFi
 ---
 
-WiFi
-====
+
+# WiFi
 
 The Raspberry Pi that makes up your kit's brain board contains a WiFi radio which allows you to interface with and debug your robot.
 You can connect to your robot using any WiFi capable device (laptop, tablet, phone, etc.)
 
-Connecting to Your Robot
-------------------------
+
+## Connecting to Your Robot
 
 During the boot process you should notice the green LED on the KCH (the board on top of the Pi) labelled "WiFi" turn on.
-Your robot has now set up its very own WiFi network! It will initially have a name starting with `robot-ZZZ` followed by some random numbers.
+Your robot has now set up its very own WiFi network!
+It will initially have a name starting with `robot-ZZZ` followed by some random numbers.
 
 You can now connect to your robot in the same way you normally connect to a WiFi network.
 You will need a WiFi key to be able to connect and you can find this inside `robot-settings.toml` on the USB drive containing your code.
 
 <div class="info">
-    If there is no <code>robot-settings.toml</code> on your USB drive, you can generate one by first running any code on your robot. For more information see our docs on <a href="{{ site.baseurl }}/kit/brain_board#robot-settings">Robot Settings</a>.
+    If there is no <code>robot-settings.toml</code> on your USB drive, you can generate one by first running any code on your robot.
```

Comment:
Nit: if you add a `markdown="1"` attribute to the `div`, you can use markdown syntax here.

Reply:
Really?
![image](https://github.com/srobo/docs/assets/25060060/2a6290d6-0afd-44d5-bbb3-7c8a8aaf9765)

Reply:
[Apparently](https://github.com/search?q=repo%3Asrobo%2Fdocs+markdown%3D%221%22&type=code), yes. Although you may need to drop the indentation

Reply:
a4aa897